app/schemas.py

```diff
@@ -105,6 +105,26 @@ class Meta:
             "_password", "verify_codes")
         strict = True
 
+    @validates('name')
+    def validate_name(self, value):
+        if not value:
+            raise ValidationError('Invalid name')
+
+    @validates('email_address')
+    def validate_email_address(self, value):
+        try:
+            validate_email_address(value)
```

Comment:
It feels like validate email should really return a true or false and then raise an exception rather than catch and rethrow.

Reply:
this is a marshmallow pattern - we have a specific error handler in `app/errors.py` that deals with `ValidationError`s

Reply:
oh wait i misread. Either way, `validate_email_address` is a utils fn and way out of scope of this PR